lib/tasks/import_data.rake

```diff
@@ -0,0 +1,41 @@
+task :import_data => :environment do
+  require 'yaml'
+
+  # Always destroy data before re-importing
+  ResourceCategory.destroy_all
+  Mindset.destroy_all
+  QuizQuestion.destroy_all
+  QuizResponse.destroy_all
+
+  # This is just to create the quiz questions below
+  ResourceCategory.create(id: 1)
```

Comment:
When I add ResourceCategory information to this script, I will obviously create more than one and put actual information in them :)